docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.0'
```

Comment:
is there a better place for this docker-compose file? feels kinda weird to have it in the root of the repo

Reply:
Agreed I forgot it was still in root, I'll tuck it away and update context 👍

Reply:
moved into examples/factorial so it's right next to the example https://github.com/honeycombio/libhoney-py/pull/167/commits/9ecb0dee76a70f3299acbc2236581475e4fab1a1

I think if/when we add more validation later with some kind of smoke test, this will likely move into its own smoke test directory.

Reply:
I think you copied it instead of moving?

Reply:
🙈 friday afternoon brain. https://github.com/honeycombio/libhoney-py/pull/167/commits/8cd360665309fff002acad8c2c6b180652253aba